src/polyfills/details/_base.scss

```diff
@@ -6,6 +6,10 @@
 summary {
 	// Make sure summary remains visible
 	display: list-item !important;
+	list: {
+		style: none;
```

Comment:
That seems not conforming according to SASS.

> ```style:``` Sets the style of the CSS output. See [Output Style](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#output_style).

http://sass-lang.com/documentation/file.SASS_REFERENCE.html#style-option